src/AppBundle/Resources/views/Report/Action/_answers.html.twig

```diff
@@ -1,7 +1,7 @@
 {% set translationDomain = "report-actions" %}
 {% trans_default_domain translationDomain %}
 {% set transOptions = {'%client%': report.client.firstname | e } %}
-{% set append104 = ('104' in report.type) ? '-104' : '' %}
+{% set append104 = ('-4' in report.type or '104' in report.type) ? '-104' : '' %}
```

Comment:
what about just 
'4' is in report.type ?
If used more than a place we'll probably need an entity method like 'is104' maybe ?